docs/administration/spaces/index.md

```diff
@@ -4,76 +4,134 @@ description: Spaces setup and configuration.
-1. Click the overflow button and select **Disable the default space**.
-1. Enter the name of the space and click **YES I'M SURE**.
+2. Expand the **Task Queue Status** section and select the Stop task queue checkbox, and click **SAVE**.
+3. Click the overflow button and select **Disable the default space**.
+4. Enter the name of the space and click **YES I'M SURE**.
 
 
-## Switching Between Spaces
+## Switching Between Spaces {#switching-between-spaces}
 
 When you log into the Octopus Web Portal, the first item on the navigation menu is the Spaces menu. Click this icon to access the spaces you are a member of and to select the space you need.
 
 ## System Scoped or Space Scoped {#system-scope-space-scoped}
 
-There is a hard barrier between Spaces, so, for instance, a deployment target configured for SpaceA isn't available to projects in SpaceB. However, there are some things that aren't scoped to a Spaces, but are system wide. <!-- content explaining why things are scoped the way they are goes here -->
+There is a hard barrier between Spaces, so, for instance, a deployment target configured for SpaceA isn't available to projects in SpaceB. However, there are some things that aren't scoped to a space, but are available system wide. <!-- content explaining why things are scoped the way they are goes here -->
 
 The following table shows what is space-scoped, system-scoped, or scoped to both.
 
-Some things can be scoped to both Spaces and the system.
-
-| Resource               | Space-Scoped                       | System-Scoped      |
-| ------------------ | --------------------------- | ---------- |
-| Environments | Spaces |  |
-| Lifecycles   | Spaces  |  |
-| Projects | Spaces |  |
-| Variable Sets | Spaces |  |
-| Deployment Targets | Spaces |  |
-| Tenants | Spaces |  |
-| Octopus Server Nodes |  | System  |
-| Authentication |  | System |
-| Users |  | System |
-| License |  | System |
-| Events | Spaces | System |
+| Resource              | Space-Scoped                | System-Scoped |
+| --------------------- | --------------------------- | ------------- |
+| Environments          | &#x2714;                    |               |
+| Lifecycles            | &#x2714;                    |               |
+| Projects              | &#x2714;                    |               |
+| Variable Sets         | &#x2714;                    |               |
+| Deployment Targets    | &#x2714;                    |               |
+| Tenants               | &#x2714;                    |               |
+| Octopus Server Nodes  |                             | &#x2714;      |
+| Authentication        |                             | &#x2714;      |
+| Users                 |                             | &#x2714;      |
+| License               |                             | &#x2714;      |
+| Events                | &#x2714;                    | &#x2714;      |
+| Teams                 | &#x2714;                    | &#x2714;      |
```

Comment:
Add Tasks to this table too

Reply:
We can add more things later. I think this list is fairly good for a starting point without us going to the effort of being exhaustive